lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -768,3 +770,137 @@ def test_discussion_tab_url(self, discussion_tab_enabled):
             assert isinstance(discussion_url, str)
         else:
             assert discussion_url is None
+
+
+@ddt.ddt
+class UserEnrollmentsStatus(MobileAPITestCase, MobileAuthUserTestMixin):
```

Comment:
Can we rename it to TestUserEnrollmentsStatus?

Reply:
Yes, of course, there was some kind of failure with the name for the test case. Changed to `TestUserEnrollmentsStatus`